projects/worker-animal-pictures/package.json

```diff
@@ -0,0 +1,41 @@
+{
+    "name": "example-worker-animal-pictures",
+    "author": "Adobe Inc.",
+    "version": "1.0.0",
+    "license": "Apache-2.0",
+    "description": "Asset Compute example custom worker - Animal Pictures",
+    "repository": "adobe/example-worker-animal-pictures",
+    "dependencies": {
+        "@adobe/httptransfer": "^1.0.1",
+        "@adobe/asset-compute-sdk": "^1.0.2"
+    },
+    "devDependencies": {
+        "@adobe/aio-cli": "^3.4.1",
+        "@adobe/eslint-config-asset-compute": "^1.3.0",
+        "cross-os": "^1.3.0"
+    },
+    "scripts": {
+        "pretest": "cross-os pretest-cross-os",
+        "test": "aio asset-compute test-worker",
+        "posttest": "cross-os posttest-cross-os"
+    },
+    "cross-os": {
+        "pretest-cross-os": {
+            "darwin": "aio plugins:install @adobe/aio-cli-plugin-asset-compute",
+            "linux": "aio plugins:install @adobe/aio-cli-plugin-asset-compute",
+            "win32": "aio plugins:install @adobe/aio-cli-plugin-asset-compute"
```

Comment:
It's identical for all OS, so the use of `cross-os` here seems completely unnecessary.

Furthermore, the `@adobe/aio-cli-plugin-asset-compute` plugin should not be installed as part of the build. We have two options, see my mail from yesterday:
1. (plan so far) make it a core plugin so it already comes with `aio`
2. (favorable new approach) make it its own cli tool and add it as `devDependency` to the projects, aka in the nui generator and in examples here

Reply:
@alexkli how would I add the plugin as a dev dependency only (even though this is only executed for tests and in a pipeline)?

Reply:
The pipeline installs all dev dependencies, isn't it? So just add the plugin in the package.json, devDependencies section. (Or did I misunderstood the discussions?).

Reply:
I was following the instructions in the AIO repo on how to install plugins, but if that works by npm installing it, I'm find with that

Reply:
@DrGirlfriend option 2 still needs work in the cli (including a rename of the repo & module, yay!) and you can't use it here yet. So for now just remove it and install `aio` + the `aio-cli-plugin-asset-compute` plugin globally in the `.travis.yml` script:

```
    npm install -g @adobe/aio-cli
    npm install -g @adobe/aio-cli-plugin-asset-compute
```